Packages.props

```diff
@@ -18,7 +18,7 @@
 	<ItemGroup Label="Test Dependencies">
 		<PackageReference Update="System.Linq.Async" Version="4.1.1" />
 		<PackageReference Update="Microsoft.NET.Test.Sdk" Version="17.0.0" />
-		<PackageReference Update="xunit" Version="2.4.1" />
+		<PackageReference Update="xunit" Version="2.4.2-pre.12" />
```

Comment:
Do we need to depend on a pre version?

Reply:
Not necessarily, but 2.4.2-pre.12 is the first version that has nullability annotations (useful for things like `Assert.NotNull` which we have in our tests.
(This doesn't change anything for `Funcky.Xunit` though. It still depends on version 2.4.0.)